src/pipecat/services/fish/tts.py

```diff
@@ -42,6 +42,7 @@ class FishAudioTTSService(InterruptibleTTSService):
     class InputParams(BaseModel):
         language: Optional[Language] = Language.EN
         latency: Optional[str] = "normal"  # "normal" or "balanced"
+        normalize: False
```

Comment:
Should be:

```suggestion
        normalize: Optional[bool] = False
```

Though, I don't see any reference to `normalize` in the websocket API docs. I only see it referenced in the HTTP API docs. I tested with it but I can't really see a difference. Can you point to the docs that confirm this parameter is supported?

Reply:
https://docs.fish.audio/text-to-speech/fine-grained-control 
The docs are a bit unclear in regards to WenSocket, when I used AskAI it recommended to set it to false.
It definitely made a difference for me, before it would read the paralanguage cues out loud, after setting it to false it applied them.

Missed the type, sorry, I'll resubmit

Reply:
I've reached out to the Fish team to confirm whether or not this is supported. In the past they've been quick to reply. Let's see what they say.

Reply:
Ok! The Fish team confirmed that normalize is supported by the websocket API.